dangerfile.js

```diff
@@ -0,0 +1,62 @@
+const execa = require('execa');
+const { fail } = require('danger');
+
+const fromRoot = path => path.replace(`${process.cwd()}/`, '');
+const codeFence = str => `\`\`\`\n${str.trim()}\n\`\`\``;
```

Comment:
Yikes. `${'```'}\n${str.trim()}\n${'```'}`, perhaps?

Maybe even set the backticks to a variable, just for readability. `${fence}\n${str.trim()}\n${fence}`

Reply:
👍Updated